deployment/bin/backup-rds

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+
+set -e
+
+# backup-data-volume /dev/xvdr
+
+RDS_ID=$(wget -q -O - http://instance-data/latest/user-data | grep 'RDS_ID=')
```

Comment:
Yes, but I will add some more explanation.

The `RDS_ID` is grepped out of the instance user data, which is a shell script that `stack.py` created.